src/test/java/com/prgrms/monthsub/module/series/series/app/SeriesServiceTest.java

```diff
@@ -284,17 +280,75 @@ void popularSort_findAllTest() {
       .sorted(comparator.reversed())
       .collect(Collectors.toList());
     doReturn(popularSeriesList)
-      .when(seriesRepository)
+      .when(this.seriesRepository)
       .findAll(any(Sort.class));
 
     //when
-    List<Series> findPopularSortList = seriesService.findAll(popularSort);
+    List<Series> findPopularSortList = this.seriesService.findAll(popularSort);
 
     //then
     verify(seriesRepository, times(1)).findAll(any(Sort.class));
-    for (int i = 0; i < 10; i++) {
-      assertThat(findPopularSortList.get(i).getId()).isEqualTo(popularSeriesList.get(i).getId());
+    assertThat(findPopularSortList).isEqualTo(popularSeriesList);
+  }
+
+  @Test
+  @DisplayName("구독 상태로 시리즈 리스트를 조회할 수 있다.")
+  void findBySubscribeStatusTest() {
+    //given
+    Long writerId = WRITER_ID;
+    Pageable pageable = PageRequest.of(
+      PAGE_NUM, SERIES_SIZE, Sort.by(Direction.DESC, "createdAt", "id"));
+    List<Series> seriesList = new ArrayList<>();
+    for (long i = 0; i < 20; i++) {
+      seriesList.add(getSeriesFixture(i, writerId));
     }
+    seriesList = seriesList.stream()
+      .filter(s -> s.getSubscribeStatus().equals(SeriesStatus.SUBSCRIPTION_AVAILABLE))
+      .sorted(Comparator
+        .comparing(Series::getCreatedAt)
+        .thenComparing(Series::getId)
+        .reversed()
+      )
+      .limit(SERIES_SIZE)
+      .collect(Collectors.toList());
+
+    doReturn(seriesList)
+      .when(seriesRepository)
+      .findBySubscribeStatus(any(SeriesStatus.class), any(Pageable.class));
+
+    //when
+    List<Series> findSeries = this.seriesService.findBySubscribeStatus(
+      SeriesStatus.SUBSCRIPTION_AVAILABLE, pageable);
```

Comment:
List<Series> findSeries = this.seriesService.findBySubscribeStatus(
      SeriesStatus.SUBSCRIPTION_AVAILABLE, pageable
);